src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,42 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+    private static final int MAX_CAPACITY = 12;
+    private static final int INVALID_STACK_VALUE = -1;
+    private static final int MAX = 11;
```

Comment:
This constant is related to MAX_CAPACITY. Changing only one leads to an error (inconsistency).